### PR TITLE
Add style choice [generic | default | custom]

### PR DESCRIPTION
**Pull request by gasiortomasz**

### CODE REVIEW

File: universal-login-react/src/ui/Notifications/EmojiPanelWithFakes.tsx [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
 
-export const EmojiPanelWithFakes = ({securityCodeWithFakes, onEmojiClicked}: EmojiPanelWithFakesProps) => {
+export const EmojiPanelWithFakes = ({securityCodeWithFakes, onEmojiClicked, className}: EmojiPanelWithFakesProps) => {
   const emojis = securityCodeWithFakes.map((code: number, index: number) => (
     <li key={`securityCodeWithFakes_${index}`} onClick={() => onEmojiClicked(code)} >
       <Emoji code={code}/>
     </li>
   ));
 
+console.log(getStyleClass(className));
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/Notifications/EmojiPanelWithFakes.tsx]: 👎

File: universal-login-react/src/core/utils/getStyleClass.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export const getStyleClass = (className?: string) => {
+  if (className === 'generic') {
+    return '';
+  }
+  else if (!className || className === 'default') {
+    return 'universal-login-default';
+  }
+  return className;
```
Comment (marekkirejczyk) [inline universal-login-react/src/core/utils/getStyleClass.ts]:
```ts
classForTopLevelComponent(className) => 
 className ? className : 'universal-login-default'
```

File: universal-login-react/src/ui/styles/emojiDefaults.css [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.universal-login-emoji ul.universal-login-default {
```
Comment (marekkirejczyk) [inline universal-login-react/src/ui/styles/emojiDefaults.css]: `.universal-login-default .universal-login-emoji ul {`

File: universal-login-react/src/ui/Notifications/EmojiPlaceholders.tsx [hunk 4]
```diff
@@ -1,21 +1,27 @@
...
     <li key={`emoji_placeholder_${index}`}  onClick={() => onEmojiClicked(index)} >
       <Emoji code={code}/>
     </li>
   ));
 
   return (
-    <ul className={'universal-login-emoji-default'}>
-      {placeholders}
-    </ul>
+    <div className={'universal-login-emoji'}>
```
Comment (marekkirejczyk) [inline universal-login-react/src/ui/Notifications/EmojiPlaceholders.tsx]: `className='universal-login-emoji'`

File: universal-login-react/src/ui/Notifications/EmojiPanelWithFakes.tsx [hunk 5]
```diff
@@ -1,24 +1,32 @@
...
+
   return (
     <div>
```
Comment (marekkirejczyk) [inline universal-login-react/src/ui/Notifications/EmojiPanelWithFakes.tsx]: Here you can use `getStyleClass`

**Pull request merged**